### ISSUE TITLE
Add "Open Recent Folder" to File menu

### ISSUE THREAD

**Issue by yonah-ag** [Jojo-Schmitz/MuseScore#448]
I use several different folders to manage libraries of related scores and it would be a nice workflow streamliner to be able to access these recent libraries directly from the File menu.

**Comment by Jojo-Schmitz** [comment]
Is that somehow related to #436?

**Comment by yonah-ag** [comment]
I don't think that 3.6.2 ever had an "Open Recent Folder" option and I can't recall whether recent paths used to be added to the recent file list. I hadn't noticed issue 436.

**Comment by zorba77** [comment]
@yonah-ag: Which is your OS? Maybe the issue #436 is related to Linux?
@Jojo-Schmitz: where are stored favorite paths? Are they managed by OS or saved in a Musescore config file?

**Comment by scorster** [comment]
> I don't think that 3.6.2 ever had an "Open Recent Folder" option ...

Here on MacOS MuseScore 3.6.2 has File>Open Recent

3.7 does as well.

**Comment by Jojo-Schmitz** [comment]
Same for me
![image](https://github.com/Jojo-Schmitz/MuseScore/assets/1786669/262dc45a-ff0d-4f03-8c71-14fd0268f905)
That is 'Recent files' though, not 'Recent folders'
